bin/cli-helpers.js

```diff
@@ -7,9 +7,26 @@ function loadVerifyExamplesConfig(configPath) {
   */
 
   var verifyExamples = null;
+  var elmJson = null;
 
   try {
     verifyExamples = require(configPath);
+    if (verifyExamples["elm-root"] === undefined) {
+      throw new Error(
+        `elm-verify-examples.json at ${configPath} does not have an elm-root key`
+      );
+    }
+    if (verifyExamples["root"] !== undefined) {
+      throw new Error(
+        `elm-verify-examples.json at ${configPath} has a root key, but it should be elm-root and point to the location of elm.json`
```

Comment:
This could be taken to mean "it should point to the elm.json file", I think?

```suggestion
        `elm-verify-examples.json at ${configPath} has a root key, but it should be elm-root and point to the root of the Elm project`
```

or

```suggestion
        `elm-verify-examples.json at ${configPath} has a root key, but it should be elm-root and point to the folder containing your elm.json`
```